index.html

```diff
@@ -438,6 +439,7 @@ <h2>
         />
```

Comment:
Please delete this stray closing tag as mentioned  by the previous reviewer, removing these is good practice and will help make your code more readable.